### PR TITLE
Hide `bot` labels in issue header

### LINKED ISSUE #5832: "Bot" label isn't dropped by `clean-conversation-headers`
### Description

Screen Shot 3

### How to replicate the issue

1. Visit https://github.com/fregante/daily-version-action/issues/22

### Extension version

22.7.1

### Browser(s) used

Safari

### PR DESCRIPTION
**Pull request by foolmoron**
Closes #5832 

## Test URLs
https://github.com/fregante/daily-version-action/issues/22

## Screenshot
image

### CODE REVIEW

**Review by fregante — COMMENTED** [review]
Thank you for the PRs! I think we can achieve this via CSS by extending the existing rule:

```diff
+ .rgh-clean-conversation-headers.rgh-clean-conversation-headers-hide-author .author + .label,
.rgh-clean-conversation-headers.rgh-clean-conversation-headers-hide-author .author {
	display: none;
}
```

We want to hide all author labels since we're already hiding the author itself.

File: source/features/clean-conversation-headers.tsx [hunk 1]
```diff
@@ -21,6 +25,17 @@ async function cleanIssueHeader(): Promise<void | false> {
 	// Removes on issues: octocat opened this issue on 1 Jan [·] 1 comments
 	const commentCount = select('relative-time', byline)!.nextSibling!;
 	commentCount.replaceWith(<span>{commentCount.textContent!.replace('·', '')}</span>);
+
+	// Remove labels with certain text contents
+	for (const label of select.all('.Label', byline)) {
+		if (!label.textContent) {
```
[the remark this answers is not in the document]
Comment (foolmoron) [reply source/features/clean-conversation-headers.tsx:31]: Sure, updated in latest

[the remark this answers is not in the document]
Comment (foolmoron) [reply source/features/clean-conversation-headers.tsx]: Oh just saw your original comment, nice. Latest uses the CSS solution.

**Comment by fregante** [comment]
Great! Thank you!

**Pull request merged**